controllers/apply/lib/field-types/file.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+const mime = require('mime-types');
+const filesize = require('filesize');
+
+const Field = require('./field');
+const Joi = require('../joi-extensions');
+
+class FileField extends Field {
+    constructor(props) {
+        super(props);
+
+        this.labelExisting = props.labelExisting || null;
+
+        const maxFileSize = {
```

Comment:
Ended up making the max size and static as file this is related to the server limits so isn't particularly variable.